src/components/canvas/Stars.js

```diff
@@ -66,6 +66,186 @@ const generateStars = lampPosts => {
       opacity: 0.9 - numLampPosts * 0.1,
       r: 1.5,
     },
+    {
+      x: 1500,
+      y: 300,
+      opacity: 1 - numLampPosts * 0.1,
```

Comment:
@NunonuN, maybe it's a good time to factor out this `0.1` to either `settings.js` under a comment `// stars`, e.g.:

```
// stars
export const STAR_OPACITY_REDUCTION_FACTOR = 0.1;
```

Or at least a constant inside the same file.

Reply:
I would then have a helper function inside the file:

```
const calculateOpacity = numLampPosts => 1 - numLampPosts * STAR_OPACITY_REDUCTION_FACTOR;
```

And then you put `calculateOpacity(numLampPosts)` on the right-hand side of all those `opacity` keys.

Reply:
This is a good idea. I will implement it.

Reply:
Done.